### ISSUE TITLE
Carthage build

### ISSUE THREAD

**Issue by niklassaers** [IBM-Swift/BlueSSLService#44]
Hi,
I make my Cartfile

```ruby
github "IBM-Swift/BlueSSLService" ~> 0.12
```
and do
```bash
carthage update
```

But the result I get is
```
Dependency "BlueSSLService" has no shared framework schemes
```

That seems fair, given that ```swift build``` doesn't build a framework, and there is no framework target in the xcodeproj. But then how do I build a BlueSocket framework with BlueSSLService included if I do not build them separately?

My motivation is that I want to make [Theo](https://github.com/GraphStory/neo4j-ios) available as a Carthage build, and it uses BlueSocket and BlueSSLService for its SSL Bolt connection.

Cheers

   Nik

**Comment by billabt** [comment]
Let me see about setting up a _framework_ target in the _xcodeproj_.  That'd be the best way...

**Comment by niklassaers** [comment]
Thanks :-)

**Comment by billabt** [comment]
Added in version _0.12.66_.  Re-open if you run into any problems.

**Comment by niklassaers** [comment]
Hi Bill,
did run into problems, but solved them and submitted PR #45 :-) Thanks for your help! :-)
